core/src/main/java/org/pentaho/platform/util/ActionUtil.java

```diff
@@ -193,6 +194,27 @@ public static String extractUid( final Map<String, Serializable> params ) {
     return uid;
   }
 
+  /**
+   * Looks up the {@code ActionUtil.USE_WORKER_NODES} within the {@link Map}. If available, the value is returned
+   *
+   *
+   * @param params a {@link Map} that may contain the {@code ActionUtil.USE_WORKER_NODES}
+   * @return {@code ActionUtil.USE_WORKER_NODES} from the {@link Map} false otherwise
+   */
+  public static boolean extractUseWorkerNodes( final Map<String, Serializable> params ) {
+    if ( params == null ) {
+      return false;
+    }
+
+    String useWorkerNodes = (String) params.get( USE_WORKER_NODES );
+    if ( StringUtil.isEmpty( useWorkerNodes ) ) {
+      return false;
+    }
```

Comment:
In cases where `useWorkerNodes` param was never passed along, then assume the default/fallback scenario.

Think for example of customers that
- have worker nodes enabled
- schedule ktrs/kjbs/prpts by calling `api/scheduler/job` endpoint directly ( i.e. not via UI )

In this case, the endpoint will remain working as always: **no `useWorkerNodes` param was ever included / passed along**; so pentaho system, if configured to do so, will leverage worker nodes capability ( whenever it's a supported `IAction` ).

So the only true impact of this param will take place when the 2 things take place:
1. `useWorkerNodes` param has been passed
2. `useWorkerNodes` was set to `false`

This translates to:
1. "normal" behaviour takes place when one of two happens:
  1.1.  `useWorkerNodes` param has been passed and is set to `true`
  1.2. `useWorkerNodes` param was never passed along
2. bypassing "normal" behaviour only takes place when:
  2.1. `useWorkerNodes` param has been passed and is set to `false`

Reply:
Closing this PR.

Reply:
Please see https://github.com/pentaho/pentaho-ee/pull/945